glide.lock

```diff
@@ -0,0 +1,33 @@
+hash: b833c4cda7c9d6b46e59553667191affe6ee81640319606aa981e58f305804d7
+updated: 2016-10-31T16:59:58.336477767-06:00
+imports:
+- name: github.com/golang/protobuf
+  version: 98fa357170587e470c5f27d3c3ea0947b71eb455
+  subpackages:
+  - proto
+- name: github.com/kmala/example-grpc-server
```

Comment:
This package and the SHA will need to change once example-grpc-server is merged.